phpstan.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+  checkMissingIterableValueType: false
+  paths:
+    - %currentWorkingDirectory%/src
```

Comment:
Missing blank line at the end of file

Reply:
Commit #b3c674c273fcac162974ee8b5ee109e637760b08